handlers/validator.go

```diff
@@ -2053,30 +2052,59 @@ func ValidatorSync(w http.ResponseWriter, r *http.Request) {
 		}
 		endIndex := start
 
-		endSlot := slots[endIndex]
-		startSlot := slots[startIndex]
+		// retrieve sync duties and sync participations
+		syncDuties := make(map[uint64]map[uint64]*types.ValidatorSyncParticipation)
```

Comment:
**suggestion:** Can this be made into a normal map? The validatorIndex "dimension" is never relevant for anything.

Reply:
Good call, fixed.